src/main/java/com/thealgorithms/bitmanipulation/GetBit.java

```diff
@@ -0,0 +1,10 @@
+package com.thealgorithms.bitmanipulation;
+/**
+ * Returns the value of bit from num located at get
+ */
+
+public class GetBit {
+    public static int getBit(int num, int get) {
+        return (num >> get) & 1;
+    }
+}

```

Comment:
In any case such functionality will be needed.

```suggestion
public final class GetBit {
    private GetBit() {
    }
    public static boolean get(final int data, final int pos) {
        if (pos < 0 || pos >= Integer.SIZE) {
            throw new ArrayIndexOutOfBoundsException();
        }
        return ((data >> pos) & 1) > 0;
    }
}
```

I am not sure what is the best way to make it accept any `IntegralType` - I could not find a convenient way how to get the `SIZE`.

Please add tests for the missing _branches_.

Reply:
Let's stick to int32 then